### PR TITLE
Add allGood section

### PR DESCRIPTION
**Pull request by KathleenQ**
If no notice is generated in the uploaded zip file, the green "All good!" is shown. It will refresh to be invisible again when a new file is uploaded.
I have manually tested it and also include the corresponding JavaScript tests.

### CODE REVIEW

**Review by KathleenQ — COMMENTED** [review]
Please ignore several lines changed due to the auto-formatting with clang format.

File: webapp/src/main/webapp/resources/output.js [hunk 1]
```diff
@@ -217,7 +221,7 @@ function same_route_name_and_description(params) {
 }
 
 function transfers_are_unique(params) {
-  const template = goog.soy.renderAsElement(
-      validator.templates.transfersAreUnique, params);
+  const template =
+      goog.soy.renderAsElement(validator.templates.transfersAreUnique, params);
```
Comment (KathleenQ) [inline webapp/src/main/webapp/resources/output.js]: Auto-formatting

File: webapp/src/main/webapp/test/output.test.js [hunk 2]
```diff
@@ -284,7 +284,8 @@ than one `agency_lang`, that\'s an error</p>\
       }]
     };
     location_without_parent_station(params);
-    const output = '<button data-toggle="collapse" data-target="#locationWithoutParentStation" class="error collapsed">Error - Location(s) without a parent station found!<span>+</span><p>-</p></button>\
+    const output =
+        '<button data-toggle="collapse" data-target="#locationWithoutParentStation" class="error collapsed">Error - Location(s) without a parent station found!<span>+</span><p>-</p></button>\
```
Comment (KathleenQ) [inline webapp/src/main/webapp/test/output.test.js]: Auto-formatting

File: webapp/src/main/webapp/test/output.test.js [hunk 3]
```diff
@@ -314,7 +315,8 @@ than one `agency_lang`, that\'s an error</p>\
       }]
     };
     meaningless_trip_with_no_more_than_one_stop(params);
-    const output = '<button data-toggle="collapse" data-target="#meaninglessTripWithNoMoreThanOneStop" class="error collapsed">Error - Meaningless trip(s) found!<span>+</span><p>-</p></button>\
+    const output =
+        '<button data-toggle="collapse" data-target="#meaninglessTripWithNoMoreThanOneStop" class="error collapsed">Error - Meaningless trip(s) found!<span>+</span><p>-</p></button>\
```
Comment (KathleenQ) [inline webapp/src/main/webapp/test/output.test.js]: Auto-formatting

File: webapp/src/main/webapp/test/output.test.js [hunk 4]
```diff
@@ -1003,14 +1008,12 @@ than one `agency_lang`, that\'s an error</p>\
...
-          csvRowNumber: 5,
-          originalCsvRowNumber: 2,
-        }
-      ]
+      notices: [{
+        fromStopId: 'fromStop',
+        toStopId: 'toStop',
+        csvRowNumber: 5,
+        originalCsvRowNumber: 2,
+      }]
```
Comment (KathleenQ) [inline webapp/src/main/webapp/test/output.test.js]: Auto-formatting

File: webapp/src/main/webapp/test/output.test.js [hunk 5]
```diff
@@ -905,7 +909,8 @@ than one `agency_lang`, that\'s an error</p>\
       ]
     };
     route_color_contrast(params);
-    const output = '<button data-toggle="collapse" data-target="#routeColorContrast" class="error collapsed">Error - Route Color Contrast!<span>+</span><p>-</p></button>\
+    const output =
+        '<button data-toggle="collapse" data-target="#routeColorContrast" class="error collapsed">Error - Route Color Contrast!<span>+</span><p>-</p></button>\
```
Comment (KathleenQ) [inline webapp/src/main/webapp/test/output.test.js]: Auto-formatting

File: webapp/src/main/webapp/test/output.test.js [hunk 6]
```diff
@@ -378,7 +381,8 @@ than one `agency_lang`, that\'s an error</p>\
       }]
     };
     overlapping_frequency(params);
-    const output = '<button data-toggle="collapse" data-target="#overlappingFrequency" class="error collapsed">Error - Overlapping frequency entries found!<span>+</span><p>-</p></button>\
+    const output =
+        '<button data-toggle="collapse" data-target="#overlappingFrequency" class="error collapsed">Error - Overlapping frequency entries found!<span>+</span><p>-</p></button>\
```
Comment (KathleenQ) [inline webapp/src/main/webapp/test/output.test.js]: Auto-formatting

File: webapp/src/main/webapp/test/output.test.js [hunk 7]
```diff
@@ -345,7 +347,8 @@ than one `agency_lang`, that\'s an error</p>\
       }]
     };
     missing_trip_edge_arrival_time_departure_time(params);
-    const output = '<button data-toggle="collapse" data-target="#missingTripEdgeStopTime" class="error collapsed">Error - Missing arrival or departure time for trip(s)!<span>+</span><p>-</p></button>\
+    const output =
+        '<button data-toggle="collapse" data-target="#missingTripEdgeStopTime" class="error collapsed">Error - Missing arrival or departure time for trip(s)!<span>+</span><p>-</p></button>\
```
Comment (KathleenQ) [inline webapp/src/main/webapp/test/output.test.js]: Auto-formatting

**Review by jupi3200 — APPROVED** [review]

**Pull request merged**